src/ai/backend/manager/api/manager.py

```diff
@@ -245,6 +245,25 @@ async def perform_scheduler_ops(request: web.Request, params: Any) -> web.Respon
     return web.Response(status=204)
 
 
+@superadmin_required
+async def scheduler_healthcheck(request: web.Request) -> web.Response:
+    root_ctx: RootContext = request.app["_root.context"]
+    raw_date = request.headers.get("Date")
+    if not raw_date:
+        raw_date = request.headers.get("X-BackendAI-Date", request.headers.get("X-Sorna-Date"))
+    if not raw_date:
+        raise GenericBadRequest
+    data = cast(dict, await root_ctx.shared_config.etcd.get_prefix("manager/scheduler"))
+    await root_ctx.event_producer.produce_event(DoScheduleEvent())
+    if len(data) == 0:
+        ret = {"event_name": "", "last_execution_time": "", "resource_group": ""}
+    else:
+        server_time = cast(str, data.get("last_execution_time"))
+        data.update({"last_execution_time": server_time[:-6] + raw_date[-6:]})
```

Comment:
is there any reason to handle `last_execution_time` like this? 
I think that we should not concat a `date` data from request header.

Reply:
Thanks!
There was some misunderstanding.
Returning the server time doesn't seem to be a problem.
I will fix it 👍